apollo-runtime-kotlin/src/commonMain/kotlin/com/apollographql/apollo/ApolloCall.kt

```diff
@@ -0,0 +1,12 @@
+package com.apollographql.apollo
+
+import com.apollographql.apollo.api.Response
+import kotlinx.coroutines.flow.Flow
+
+interface ApolloCall<T> {
+  fun execute(): Flow<Response<T>>
+}
+
+interface ApolloQueryCall<T> : ApolloCall<T>
+
+interface ApolloMutationCall<T> : ApolloCall<T>
```

Comment:
I have two somewhat contradictory questions: 
- Why are we using multiple interfaces here if they're not adding any functionality?
- Why isn't there an `ApolloSubscriptionCall<T>`?

Reply:
1. it's for the future like normalized cache, any API for normalized cache will be defined as an extension functions, but we need to have 2 separate interfaces as only for cache is supported for `ApolloQueryCall`

2. `ApolloSubscriptionCall`  will be added when we start working on subscription KN version.

Reply:
I suspect people will want mutation results to update the cache as well, but that makes sense.